worlds/sc2/Client.py

```diff
@@ -530,22 +537,22 @@ def __init__(self, *args, **kwargs) -> None:
         self.location_inclusions: typing.Dict[LocationType, int] = {}
         self.plando_locations: typing.List[str] = []
         self.current_tooltip = None
-        self.last_loc_list = None
         self.difficulty_override = -1
         self.game_speed_override = -1
         self.mission_id_to_location_ids: typing.Dict[int, typing.List[int]] = {}
         self.last_bot: typing.Optional[ArchipelagoBot] = None
         self.slot_data_version = 2
-        self.grant_story_tech = 0
-        self.required_tactics = RequiredTactics.option_standard
-        self.take_over_ai_allies = TakeOverAIAllies.option_false
+        self.required_tactics: int = RequiredTactics.default
+        self.grant_story_tech: int = GrantStoryTech.default
+        self.grant_story_levels: int = GrantStoryLevels.default
+        self.take_over_ai_allies: int = TakeOverAIAllies.default
         self.spear_of_adun_presence = SpearOfAdunPresence.option_not_present
         self.spear_of_adun_present_in_no_build = SpearOfAdunPresentInNoBuild.option_false
         self.spear_of_adun_autonomously_cast_ability_presence = SpearOfAdunAutonomouslyCastAbilityPresence.option_not_present
         self.spear_of_adun_autonomously_cast_present_in_no_build = SpearOfAdunAutonomouslyCastPresentInNoBuild.option_false
-        self.minerals_per_item = 15
-        self.vespene_per_item = 15
-        self.starting_supply_per_item = 2
+        self.minerals_per_item: int = MineralsPerItem.default
+        self.vespene_per_item: int = VespenePerItem.default
+        self.starting_supply_per_item: int = StartingSupplyPerItem.default
```

Comment:
These values were set so in order to keep the previously generated games intact (0.4.4 games and older)

Reply:
Reverted. I don't think there's very many ongoing pre-0.4.5 games, so it's probably safe to update it in a few weeks' time.